### ISSUE TITLE
Adjust element tag of blocks or block items

### ISSUE THREAD

**Issue by spencer-j** [nk-o/ghostkit#50]

**Comment by nk-o** [comment]
Hi.

It is a good addition, but I'm not sure Free plugin will get these options since it is a bit confusing for users to choose from so many options and see no difference. I think it is better to place in the PRO addon. I will keep it in mind, thank you.

Regards, nK.

**Comment by spencer-j** [comment]
Thats right, maybe even better if some options/ fields of block settings could be activated/deactivatit in ghostscript settings... maybe per backend user roles.
Best regards spencer
